dts/arm/st/l4/stm32l433Xb.dtsi

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
@KwonTae-young is this header included by any in-tree Board, SoC?

Reply:
As far as I know, all *.dtsi contain these headers.